src/unix/stream.c

```diff
@@ -698,7 +698,8 @@ static int uv__write_req_update(uv_stream_t* stream,
 
   do {
     len = n < buf->len ? n : buf->len;
-    buf->base += len;
+    if (buf->len != 0)
```

Comment:
This seems unnecessary

Reply:
You can't increment a null pointer, even by 0.

Reply:
This is one of the fixes for UBSAN failures.